# Generated by Django 3.1.2 on 2021-01-14 11:52

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('account_post', '0005_accountpost_error_happened_on_uploading_image'),
    ]

    operations = [
        migrations.AddField(
            model_name='accountpost',
            name='is_url_valid',
            field=models.BooleanField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='accountpost',
            name='url',
            field=models.URLField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='accountpost',
            name='url_action_text',
            field=models.CharField(blank=True, max_length=10, null=True),
        ),
    ]
